### ISSUE TITLE
BURP Proxy not working on https connections.

### ISSUE THREAD

**Issue by josarsepi** [FelisCatus/SwitchyOmega#1040]
I'm having an issue getting this proxy to work with https requests using the burp proxy.  Has anyone had any luck getting that to work?

HTTP is working correctly.

HTTPS connections work with other proxies (i.e. SOCKS5 via SSH tunneling).

**Comment by celeron533** [comment]
https://github.com/FelisCatus/SwitchyOmega/issues/845#issuecomment-243644157
Chrome 51 do not support wildcard in https URL any more.

**Comment by josarsepi** [comment]
I'm not sure this is the issue.  Specifically, https://www.google.com will work with something like the SSH tunneling but not with the BURP proxy.

**Comment by FelisCatus** [comment]
Can you provide a screenshot of the proxy servers section please? You may change or mask the server address to protect your privacy.

Also, for most of the time, only the `(default)` row should be filed. If it's not the case, try clearing all settings in other row. If your proxy has specific instructions, please follow that instruction.

If you think it is an issue with the network or proxy itself, please report to the proxy's maintainer. SwitchyOmega is only a configuration tool.

**Comment by josarsepi** [comment]
Thanks for looking at this Felis.  I looked at my configuration and began prepping my screenshots for you and it appears I must have mis-configured something yesterday.  BURP is now working with the settings shown below.  I'm sorry if I wasted your time.

![capture120](https://cloud.githubusercontent.com/assets/6023519/24673927/d6ac1974-1947-11e7-8471-4fa5e311ad36.PNG)

-----

![capture121](https://cloud.githubusercontent.com/assets/6023519/24673926/d6ab5f0c-1947-11e7-86ba-3f8979a8c986.PNG)
